flutter/lib/consts.dart

```diff
@@ -79,6 +79,7 @@ const String kWindowEventOpenMonitorSession = "open_monitor_session";
 
 const String kOptionViewStyle = "view_style";
 const String kOptionScrollStyle = "scroll_style";
+const String kOptionEdgeScrollEdgeThickness = "edge_scroll_edge_thickness";
```

Comment:
It is recommended to change it to `edge-scroll-edge-thickness`, as the use of underscores is for historical reasons.

Reply:
That feels weird directly adjacent to `view_style` and `scroll_style`, but I can of course make that change. 😛